docs/process/bazel.md

```diff
@@ -58,6 +59,14 @@ causes jasmine to exit with a non-zero exit code.
 
 While testing, you can remove the `shard_count` attribute to prevent sharding and the `flaky`
 attribute to prevent repetition.
+Setting `--test_output=streamed` will disable sharding and `--flaky_test_attempts=1` will disable
```

Comment:
Note: you can also use the `no-sharding` config which disables sharding and sets `flaky_test_attempt` to 1

Reply:
Added a mention of `no-sharding` to the example command.